src/lib.rs

```diff
@@ -57,7 +57,7 @@
 #![deny(unused_imports)]
 #![deny(missing_docs)]
 #![deny(unused_must_use)]
-#![deny(broken_intra_doc_links)]
+#![deny(rustdoc::broken_intra_doc_links)]
```

Comment:
Hmm, looks like CI is failing. Could be this only works because of my Rust version:
```shell
rustc 1.61.0 (fe5b13d68 2022-05-18)
```
@Kixunil feel free to close this if you think the CI is correct.